projects/components/src/multi-select/multi-select.component.ts

```diff
@@ -218,6 +218,7 @@ export class MultiSelectComponent<V> implements ControlValueAccessor, AfterConte
 
   public filteredOptions$!: Observable<SelectOptionComponent<V>[]>;
   private readonly caseInsensitiveSearchSubject: BehaviorSubject<string> = new BehaviorSubject('');
+  private readonly reorderSelectedItemsSubject: BehaviorSubject<boolean> = new BehaviorSubject<boolean>(false);
```

Comment:
not required

Reply:
I couldn't find a way to force the combineLatests without to use this behaviorSubject